app/views/sciruby_github/repos.erb

```diff
@@ -2,18 +2,38 @@
 <div class="span12 pagination-centered" style="text-align: center;">
     <h3> SciRuby github data analysis (Using daru-view/googlecharts)</h3>
     <b>Data is taken from url : https://api.github.com/orgs/Sciruby/repos </b>
-    <%= render 'export' %>
+    <%=
+      daru_download formats: %w[csv xls json], separator: '|'
+    %>
     <p style="left: 50%">
       <b>SciRuby Datatable : </b>
-      <%=raw @df_repo_table.div %>
+      <%=
+        daru_table @df_repos,
+                   adapter: :googlecharts,
+                   pageSize: 10,
+                   height: 300,
+                   width: 400
```

Comment:
I believe (considering how small time we have left) that just doing the "most popular" thing (e.g., for now, just add helper for Rails, to showcase everything properly) would be enough for this week ;)